### PR TITLE
[11] - Get coin usd value by id

### LINKED ISSUE #11: Como desarrollador quiero integrar una API externa para obtener informacion de las criptomonedas
**Description**
Como desarrollador quiero integrar llamadas a una API externa para obtener la información de las criptomonedas

### PR DESCRIPTION
**Pull request by xAriextz**
Review for issue #11

### CODE REVIEW

**Review by jesuslezaun — APPROVED** [review]
Ok to merge

File: app/Application/CryptoDataSource/CoinLoreCryptoDataSource.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    public function findCoinById(string $coinId): Coin
+    {
+        $genericCoinData = $this->getGenericCoinData($coinId);
+        return new Coin(
+            $genericCoinData[0]->id,
+            $genericCoinData[0]->name,
+            $genericCoinData[0]->symbol,
+            0,
```
Comment (raulsc-cl) [inline app/Application/CryptoDataSource/CoinLoreCryptoDataSource.php]: Create a variable for amount of coin

**Review by raulsc-cl — CHANGES_REQUESTED** [review]
Make the requested changes

**Review by raulsc-cl — APPROVED** [review]
Okey to merge

**Review by jesuslezaun — APPROVED** [review]
Okay to merge

**Review by raulsc-cl — APPROVED** [review]
Okey to merge

**Pull request merged**